### ISSUE TITLE
Add ability of customization for karaoke mode window

### ISSUE THREAD

**Issue by async3619** [aviwad/LyricFever#70]
It would be good if LyricsFever supports to customize karaoke mode window (background color, font-size, font-family, etc.) like how LyricsX worked

**Comment by aviwad** [comment]
You're right this is a big feature missing right now. 

Background color: manually set one or use the Spotify color for the song
Font size: set a number
Font family: pick a font (potentially upload your own font file?)

anything else?

**Comment by async3619** [comment]
It can be nice if we could control background transparency of lyrics window tbh

**Comment by aviwad** [comment]
Ok noted. (Just curious, would you reduce or increase the transparency?)

**Comment by async3619** [comment]
> Just curious, would you reduce or increase the transparency?

I didn't understand what you mean, Is there any feature to reduce/increase transparency? If there isn't, I would use it fr

**Comment by qqaatw** [comment]
Thank you for your work - simple but useful! It would be great to have customizations for the karaoke mode.

**Comment by Hurdys** [comment]
Karaoke window pops up on the border of my screen and I can't drag it around..

**Comment by Hurdys** [comment]
I agree. The possibility to make the background transparent would be really great. Or if would be possible for the karaoke window to fade invisible when my cursor is hovering over it.

even without these features, thanks for this project!

**Comment by aviwad** [comment]
[Lyric Fever.zip](https://github.com/user-attachments/files/18936988/Lyric.Fever.zip)

Please test drive version 2.1 😄. Do note that once the official update is out you will manually have to delete this one so maybe just keep this in your Downloads folder. Thank you, any feedback would be great.

**Comment by aviwad** [comment]
Please update to the released version 2.1. You can trash the beta file I distributed and download a fresh copy of the dmg here: https://github.com/aviwad/LyricFever/releases/tag/v2.1

**Comment by akulgoel96** [comment]
thank you so much!
